src/handleUrl.js

```diff
@@ -0,0 +1,32 @@
+import globby from 'globby';
+
+/**
+ * handle globby filepath and return an array with all matched assets.
+ *
+ * @export
+ * @param {Array} assets
+ * @returns
+ */
+export default async function(assets) {
+  const globbyAssets = [];
+  const normalAssets = [];
+  // if filepath is null or undefined, just bubble up.
+  assets.forEach(
+    asset =>
+      asset.filepath && globby.hasMagic(asset.filepath)
+        ? globbyAssets.push(asset)
+        : normalAssets.push(asset)
+  );
+  const ret = [];
+  await Promise.all(
```

Comment:
Rewrote this a bit, thoughts?

https://github.com/SimenB/add-asset-html-webpack-plugin/pull/78/commits/4ef0ba202ef35c3aec69babed83a3bba1fb05f79